src/AFL-3.0.xml

```diff
@@ -0,0 +1,208 @@
+<spdx name="Academic Free License v3.0" identifier="AFL-3.0" osi-approved="true">
+  <urls>
+    <url>http://www.rosenlaw.com/AFL3.0.htm</url>
```

Comment:
> rosenlaw link appears to no longer work

This issue still stands, and there's still a needs-edit label on this PR, so I'm not sure why it was merged.  Should I file a PR removing the broken link?